### ISSUE TITLE
Summon idris2 with `rlwrap`?

### ISSUE THREAD

**Issue by andrevidela** [kvwatkins/Idris2-repl#1]
currently the idris2 repl does not support arrow key movement, but we can use `rlwrap` instead, is there a way to alias `idris2` with `rlwrap idris2`?

**Comment by kvwatkins** [comment]
Good catch. I just pushed a fix for this, on load it will be rlwraped. I had some issues trying to get an alias to stick on startup to truly alias it so if the user :q out of the idris session and re summons, the keys will be broken. Will keep the issue open until that minor correction is fixed.
